src/test/kotlin/net/atos/zac/app/ontkoppeldedocumenten/OntkoppeldeDocumentenRESTServiceTest.kt

```diff
@@ -0,0 +1,202 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+package net.atos.zac.app.ontkoppeldedocumenten
+
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.core.spec.IsolationMode
+import io.kotest.core.spec.style.BehaviorSpec
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.shouldNotBe
+import io.mockk.clearAllMocks
+import io.mockk.every
+import io.mockk.just
+import io.mockk.mockk
+import io.mockk.runs
+import io.mockk.verify
+import net.atos.client.zgw.drc.DrcClientService
+import net.atos.client.zgw.drc.model.generated.EnkelvoudigInformatieObject
+import net.atos.client.zgw.shared.exception.FoutException
+import net.atos.client.zgw.shared.model.Fout
+import net.atos.client.zgw.zrc.ZRCClientService
+import net.atos.client.zgw.zrc.model.ZaakInformatieobject
+import net.atos.zac.app.identity.converter.RESTUserConverter
+import net.atos.zac.app.ontkoppeldedocumenten.converter.RESTOntkoppeldDocumentConverter
+import net.atos.zac.app.ontkoppeldedocumenten.converter.RESTOntkoppeldDocumentListParametersConverter
+import net.atos.zac.documenten.OntkoppeldeDocumentenService
+import net.atos.zac.documenten.model.OntkoppeldDocument
+import net.atos.zac.policy.PolicyService
+import net.atos.zac.policy.output.createWerklijstRechten
+import java.net.URI
+import java.util.Optional
+import java.util.UUID
+
+class OntkoppeldeDocumentenRESTServiceTest : BehaviorSpec({
+    val ontkoppeldeDocumentenService = mockk<OntkoppeldeDocumentenService>()
+    val drcClientService = mockk<DrcClientService>()
+    val zrcClientService = mockk<ZRCClientService>()
+    val ontkoppeldDocumentConverter = mockk<RESTOntkoppeldDocumentConverter>()
+    val listParametersConverter = mockk<RESTOntkoppeldDocumentListParametersConverter>()
+    val userConverter = mockk<RESTUserConverter>()
+    val policyService = mockk<PolicyService>()
+
+    val sut = OntkoppeldeDocumentenRESTService(
```

Comment:
`sut` is a bit cryptic and requires you to look at the title of the file to get up-to-speed. Wouldn't it be better to have this names something like this:
```suggestion
    val disconnectedDocumentsRestService = OntkoppeldeDocumentenRESTService(
```